know/rdf/parser/sparql_reader.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import os
+
+
+def read(query_file, uri):
```

Comment:
You need to rename this to like `read_sparql` or something, since `read` is a reserved function in Python (like `open` or `print`: https://www.w3schools.com/python/ref_file_read.asp)